api/src/models/JobMetadata.ts

```diff
@@ -83,30 +99,33 @@ class JobMetadata {
         this._documents = DocumentOrder.fromRaw(data);
     }
 
-    get raw() {
-        return {
-            order: this.order.raw,
-            published: this.published  
-        };
+    get audio() {
+        if (this._audio === null) {
+            this._audio = AudioOrder.fromJob(this._audio);
```

Comment:
I think this should be `fromJob(this.job)`, not `fromJob(this._audio)`.

Reply:
Looks like the documents() method above has the same bug (or a different variation thereof).